wagtail/contrib/postgres_search/apps.py

```diff
@@ -1,13 +1,20 @@
+import warnings
+
 from django.apps import AppConfig
 from django.core.checks import Error, Tags, register
 
+from wagtail.utils.deprecation import RemovedInWagtail216Warning as RemovedInWagtail217Warning
+
 from .utils import get_postgresql_connections, set_weights
 
 
 class PostgresSearchConfig(AppConfig):
     name = 'wagtail.contrib.postgres_search'
 
     def ready(self):
+
+        warnings.warn('The postgres_search app will be deprecated in a future release. Please update your search backend to \'wagtail.search.backends.database\', which will pick up a Postgres backend automatically.', RemovedInWagtail217Warning)
```

Comment:
I think we should change ``postgres_search`` to ``'wagtail.contrib.postgres_search'`` since this is how it would be registered in settings files

Reply:
Changed! 🙏🏻